openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1049,7 +1049,7 @@ If the Wallet is requesting the issuance of a Credential that is not supported b
   * `invalid_credential_request`: The Credential Request is missing a required parameter, includes an unsupported parameter or parameter value, repeats the same parameter, or is otherwise malformed.
   * `unsupported_credential_type`: Requested Credential type is not supported.
   * `unsupported_credential_format`: Requested Credential Format is not supported.
-  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid. For example, because either both fields are missing or both are present simultaneously, or one of the provided key proofs is either invalid or not linked to a nonce provided by the Credential Issuer.
+  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid. For example, this could be because either both fields are missing, both are present simultaneously, one of the provided key proofs is either invalid or if a `c_nonce` was provided by the Credential Issuer, one of the key proofs is not linked to a previously issued valid `c_nonce` value (see (#issuer-provided-nonce)).
```

Comment:
this sentence reads a bit weird to me especially the part

> ..., one of the provided key proofs is either invalid or if a `c_nonce` was provided by the Credential Issuer,...

Reply:
@nemqe can you check whether it is more readable now